packages/ripple-nuxt-tide/README.md

```diff
@@ -2,7 +2,180 @@
 
 > The Nuxt module for adding Tide integration.
 
-## Setup
+## Nuxt configuration
+
+A `nuxt.config.js` file should exist in the root of your nuxt-tide site.
+
+The basic anatomy of a `nuxt.config.js` file is:
+
```

Comment:
These are the doc for Nuxt.js, I don't think it's our module's scope.

Reply:
Removed.